app/views/participants/_table.haml

```diff
@@ -8,23 +8,25 @@
         %th
         %th
     %tbody
-      = form_for Participant.new do |f|
-        %tr
+      = form_with model: Participant.new, local: true do |f|
+        %tr#new_participant
           %td= f.text_field :name, placeholder: 'Name', class: 'form-control'
           %td= f.number_field :number, class: 'form-control'
           %td= f.collection_select :bus_id, @buses, :id, :number,
             {include_blank: true}, {class: 'form-control'}
           %td= f.submit 'Add', class: 'btn btn-primary'
           %td
       - @participants.each do |participant|
-        %tr
-          = form_for participant do |f|
+        %tr{ data: { participant_id: participant.id } }
+          = form_with model: participant, namespace: participant.id, local: true do |f|
```

Comment:
View fixes make the tests a bit easier, but also fix the fact that every participant name box on this page had the same id